### PR TITLE
WAZO-2259: update plugins metadata

### PR DESCRIPTION
**Pull request by bloom1**
ALL plugins will need to be rebuilt & repushed on stable after this.

### CODE REVIEW

File: plugins/xivo-aastra/3.3.1-SP4/plugin-info [hunk 1]
```diff
@@ -8,7 +8,8 @@
             "high_availability": true,
             "function_keys": 8,
             "expansion_modules": 0,
-            "switchboard": false,
+            "switchboard": false
+            "type": "deskphone",,
```
Comment (sboily) [inline plugins/xivo-aastra/3.3.1-SP4/plugin-info:12]: Double comma

File: plugins/wazo-snom/8.7.5.35/plugin-info [hunk 2]
```diff
@@ -113,6 +124,7 @@
             "function_keys": 0,
             "expansion_modules": 0,
             "switchboard": false,
+            "type": "bridge",
```
Comment (sboily) [inline plugins/wazo-snom/8.7.5.35/plugin-info:127]: What is bridge? It's a pager.

**Comment by bloom1** [comment]
recheck

File: plugins/xivo-aastra/3.3.1-SP4/plugin-info [hunk 3]
```diff
@@ -33,14 +36,16 @@
...
             "protocol": "sip"
         },
         "Aastra, 6739i, 3.3.1.4365": {
             "lines": 9,
             "high_availability": true,
             "function_keys": 55,
             "expansion_modules": 0,
-            "switchboard": false,
+            "switchboard": false
+            "type": "deskphone",,
```
Comment (sboily) [inline plugins/xivo-aastra/3.3.1-SP4/plugin-info:48]: Double comma

**Review by sboily — CHANGES_REQUESTED** [review]

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
I'm not sure what to review, according the ticket all plugin should implement keys defined in the readme

The funny command to find them:
```shell
find -name plugin-info -exec jq --arg filename {} '.capabilities[] | select( has("type") == false ) | $filename' {} \;
```

high_availability is missing:
* xivo-cisco-sccp/wireless-1.4.5/plugin-info
* xivo-cisco-sccp/9.4/plugin-info
* xivo-cisco-sccp/cipc-2.1.2/plugin-info
* xivo-cisco-sccp/8.5.2/plugin-info
* xivo-cisco-sccp/legacy/plugin-info

expansion_modules is missing:
* wazo-fanvil/serie-x/plugin-info

switchboard is missing:
* ./xivo-zenitel/01.11.3.2/plugin-info
* xivo-grandstream/1.0.8.6/plugin-info
* ./wazo-fanvil/serie-x/plugin-info
* ./wazo-fanvil/2.3/plugin-info
* ./wazo-htek/2.0.4.4.58/plugin-info
* ./wazo-htek/2.0.4.6.41/plugin-info
* ./xivo-cisco-sccp/wireless-1.4.5/plugin-info
* ./xivo-cisco-sccp/9.4/plugin-info
* ./xivo-cisco-sccp/cipc-2.1.2/plugin-info
* ./xivo-cisco-sccp/8.5.2/plugin-info
* ./xivo-cisco-sccp/legacy/plugin-info
* ./xivo-technicolor/TB30-1.74.0/plugin-info
* ./xivo-technicolor/ST2022-4.78.1/plugin-info
* ./xivo-technicolor/ST2030-2.74/plugin-info
* ./xivo-yealink/v73/plugin-info
* xivo-yealink/v85/plugin-info
* ./xivo-yealink/v80/plugin-info
* ./xivo-yealink/v83/plugin-info
* ./xivo-yealink/v82/plugin-info
* ./xivo-yealink/v84/plugin-info
* ./xivo-yealink/v81/plugin-info

type is misisng:
* "./xivo-grandstream/1.0.8.6/plugin-info"
* "./wazo-snom/8.7.5.35/plugin-info"
* "./wazo-snom/8.9.3.80/plugin-info"
* "./core/zero/plugin-info"
* "./core/null/plugin-info"

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**